src/app/components/AppNavigator/TabIcon/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Image, ImageSourcePropType } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  focused: boolean;
+  source: ImageSourcePropType;
+}
+function TabIcon({ focused, source }: Props) {
+  return <Image style={focused ? styles.imageFocused : styles.imageUnFocused} source={source} />;
```

Comment:
Use route hook to get the route name and Icon:
```
...
  const route = useRoute();
  const ICON = NAV_ICONS[route.name];
  return (
    <Image source={ICON} style={focused ? styles.imageFocused : styles.imageUnFocused} />
  );
...
```
And NAV_ICONS object is:
```
export const NAV_ICONS = {
  Library: libraryImage,
  Wishlist: wishlistImage,
  AddNew: addNewImage,
  Rentals: rentalsImage,
  Settings: settingsImage
};
```
Use constant for each Route name (from `@constants/routes`